libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/GranteeList.tsx

```diff
@@ -25,10 +26,11 @@ export const GranteeList: React.FC<IGranteesListProps> = (props) => {
 
     return (
         <div className="gd-share-dialog-grantee-list">
-            {sortedGrantees.map((grantee) => {
+            {sortedGrantees.map((grantee, i) => {
                 return (
                     <GranteeItemComponent
-                        key={serializeObjRef(grantee.id)}
+                        currentUserPermissions={currentUserPermissions}
+                        key={serializeObjRef(grantee.id) + i} // TODO: remove index when mock data is removed from BE
```

Comment:
Remove index from key

Reply:
Mocked data is no longer returned from BE. Removed.